internal/cluster/distribution/pke/pkeaws/pkeawsworkflow/workflow_update_node_pool.go

```diff
@@ -0,0 +1,78 @@
+// Copyright © 2020 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pkeawsworkflow
+
+import (
+	"fmt"
+
+	"go.uber.org/cadence/workflow"
+
+	"github.com/banzaicloud/pipeline/internal/cluster"
+	"github.com/banzaicloud/pipeline/internal/cluster/distribution/pke"
+)
+
+const UpdateNodePoolWorkflowName = "pkeaws-update-node-pool"
```

Comment:
**Note**: FYI at the PKE Azure and vSphere implementation we used `pke-azure-<activity>` and `pke-vsphere-<activity>` activity names. Not sure if we want to do it that way here as well or use the `pkeaws-`, `pke-aws-`, `pkeamazon` or `pke-amazon`, just wanted to point it out and let you think about it, I'm unsure about what would be best.

Reply:
I followed mostly the EKS pattern here, so PKE AWS namings are now a mix of EKS and general PKE and some PKE AWS specific namings. Let me check this! :)

Reply:
TBH `pke-aws-` looks much better to me (and we have `pke-azure-`, and `pke-vsphere-` already), so I will rename it like that if you are okay with ith.

Reply:
That sounds fine.